docs/source/contributing.rst

```diff
@@ -66,22 +82,22 @@ We use `pytest <https://docs.pytest.org/en/latest/>`_ for unit testing. To execu
 
 .. code-block:: bash
 
-    make qt
+    $ make qt
 
 or for a specific file:
 
 .. code-block:: bash
 
-    py.test path/to/file
+    $ py.test path/to/file
 
 If you want to measure test coverage you can run:
 
 .. code-block:: bash
 
-    make coverage
+    $ make coverage
 
 Wagtailtrans supports multiple environments which can be tested with ``tox``. It takes a bit longer to complete, but you can run it by a simple command:
 
 .. code-block:: bash
 
-    tox
+    $ tox
```

Comment:
By running just ``tox`` you should have a setup with multiple python versions, perhaps this is a bit too much to have explained in the documentation, maybe a blog post we can link to?